### ISSUE TITLE
training error

### ISSUE THREAD

**Issue by zwb170010** [liudakai2/UnsupDIS-pytorch#1]
Hello, when I run on dataset of myself, there is no error message reported during the test phase of the training code, why does the program of end automatically
![Screenshot from 2022-05-11 16-30-55](https://user-images.githubusercontent.com/73511206/167806477-fc5855d8-cad9-488c-93b1-0b65bb043e5b.png)

**Comment by liudakai2** [comment]
It may be caused by the line 96 in test.py:

                    `check_align_output(*pred[1:], _exit=True)`

Since the output images/masks are all zeros in the testing phase.

**Comment by FXY0117** [comment]
Excuse me ,when I training on Warped COCO dataset, it shows this errors:
![image](https://user-images.githubusercontent.com/74040707/178442424-e56a5618-7f05-4a9e-8ee1-151394e9fe20.png)

Is there something wrong when dealing with the dataset?

**Comment by FXY0117** [comment]
Excuse me, I have checked the dataset path, it is all right. What's more, is there only input1's label in train.txt and test.txt?
May it is because the mismatch between labels and images？
![image](https://user-images.githubusercontent.com/74040707/178672710-5b88acb5-c542-45e3-9ad3-293a2ce155be.png)

**Comment by liudakai2** [comment]
> Excuse me, I have checked the dataset path, it is all right. What's more, is there only input1's label in train.txt and test.txt? May it is because the mismatch between labels and images？ ![image](https://user-images.githubusercontent.com/74040707/178672710-5b88acb5-c542-45e3-9ad3-293a2ce155be.png)

1. The unsupervised image stitching task has no labels. The labels of synthetic WarpedCOCO are located at WarpedCOCO/training/shift/*.npy and WarpedCOCO/testing/shift/*.npy.
2. Can you execute the following command in your project directory and show the results?
`ls WarpedCOCO/testing/input1/ | head -10`
It should return like this:
![image](https://user-images.githubusercontent.com/50127983/178681098-44e98929-5882-4434-b68c-188928668f75.png)

**Comment by FXY0117** [comment]
Ok, it returns right.
![image](https://user-images.githubusercontent.com/74040707/178683869-eedfa405-d025-44b3-8d42-e5d2bf31ee60.png)

**Comment by liudakai2** [comment]
> Ok, it returns right. ![image](https://user-images.githubusercontent.com/74040707/178683869-eedfa405-d025-44b3-8d42-e5d2bf31ee60.png)

The `WarpedCOCO` directory should be located in your `UnsupDIS-pytorch-master/WarpedCOCO` rather than `UnsupDIS-pytorch-master/data/WarpedCOCO`.

**Comment by FXY0117** [comment]
Oh，so careless am I, it works, thank you very much

**Comment by liudakai2** [comment]
> Oh，so careless am I, it works, thank you very much

Glad to hear that. We will also release the pre-trained models soon (maybe this weekend or next week). Have fun with this repo!

**Comment by FXY0117** [comment]
Sorry to bother you again, when finetuning align model on UDIS-D dataset, it breaks because that:
![image](https://user-images.githubusercontent.com/74040707/178899743-01c1f8eb-be2f-4696-b57b-7445ad350f15.png)
I placed UDIS-D dataset like WarpCOCO

**Comment by liudakai2** [comment]
> Sorry to bother you again, when finetuning align model on UDIS-D dataset, it breaks because that: ![image](https://user-images.githubusercontent.com/74040707/178899743-01c1f8eb-be2f-4696-b57b-7445ad350f15.png) I placed UDIS-D dataset like WarpCOCO

Maybe you can print and check the path of the missing image.

**Comment by FXY0117** [comment]
> > Sorry to bother you again, when finetuning align model on UDIS-D dataset, it breaks because that: ![image](https://user-images.githubusercontent.com/74040707/178899743-01c1f8eb-be2f-4696-b57b-7445ad350f15.png) I placed UDIS-D dataset like WarpCOCO
> 
> Maybe you can print and check the path of the missing image.

Ok, I have checked the image, it seems right. But I don't know how to print the missing image of you said. What's more,  is the UDIS dataset you trained same as official implementation with tensorflow . I downloaded the dataset from there, and I suggest you can add the BaiduCloud link in your git. Thank you for your solution!

**Comment by liudakai2** [comment]
Maybe you can print and check the path of the missing image.



---Original---
From: "See ***@***.***&gt;
Date: Thu, Jul 14, 2022 12:39 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;"Liu ***@***.***&gt;;
Subject: Re: [liudakai2/UnsupDIS-pytorch] training error (Issue #1)




 
Sorry to bother you again, when finetuning align model on UDIS-D dataset, it breaks because that:
 
 I placed UDIS-D dataset like WarpCOCO
